src/core/gltf2importer/gltf2importer.cpp

```diff
@@ -208,13 +209,23 @@ using namespace Kuesa;
     \brief if true, assets with no names will be added to collections with default names (default is false)
  */
 
+/*!
+ * \property GLTF2Importer::options
+ * \brief Holds the Kuesa::GLTF2Import::GLTF2Options used to select what to import/generate
```

Comment:
Was this indented in QtC with Ctrl + E, R ?

Reply:
Does nothing. I can indent it by hand so it follows the properties above